data/sql/updates/pending_db_world/rev_1731793823275142100.sql

```diff
@@ -0,0 +1,132 @@
+--
+SET @GUID := 45537;
```

Comment:
May I ask why this SAI is GUID based and not template based?

Reply:
Because 1 of them is active patrolling the starting area while all others are static

Reply:
> Because 1 of them is active patrolling the starting area while all others are static

Is it possible to have the template assigned SAI and have the GUID override it?

Reply:
I think I got it